client/lib/plans/index.js

```diff
@@ -566,3 +568,13 @@ export const chooseDefaultCustomerType = ( { currentCustomerType, selectedPlan,
 
 	return 'personal';
 };
+
+/**
+ * Determines if a plan includes Jetpack Search by looking at the plan's features.
+ *
+ * @param   {string}  planSlug  Slug of the plan.
+ * @returns {boolean}           Whether the specified plan includes Jetpack Search.
+ */
+export const hasPlanJetpackSearch = ( planSlug ) =>
```

Comment:
Could we rename this function for clarity? It's not obvious what `hasPlanJetpackSearch` means. 

Perhaps something like `planHasJetpackSearch`? Or even `hasJetpackSearch`?

Reply:
I concur that the name might be confusing. I'll change it to `planHasJetpackSearch`. Just for context, the structure of the name came from a naming pattern, I think, we implicitly follow in Calypso (I could be totally wrong because I just have 7 months or so of experience working on this codebase). Here some examples I found by searching for `hasSite`:

![image](https://user-images.githubusercontent.com/3418513/94964300-f62fc800-04cf-11eb-8582-7048254ce812.png)

Just out of curiosity, why do you think the name is not obvious? Is it because it's unusual to say that a plan _has_ a product? Would it be better if we talk about Jetpack Search as a feature? That is, at least, how these two products are related in Calypso's codebase. We say that Jetpack Search is a hidden feature of Jetpack Complete.

In any case, as I said before, I'll change it to `planHasJetpackSearch`. Thank you again for review.

Reply:
> Just out of curiosity, why do you think the name is not obvious?

I think `hasPlanJetpackSearch` could be interpreted in two different ways: 

- Having a plan named Jetpack Search
- Having a plan with Jetpack Search

The latter reading works perfectly for our use-case here, but the former is completely off the mark :)

Reply:
> Having a plan named Jetpack Search

Never thought about this one and I can definitely see people interpreting it like that. Now agree 100% with you.

(I just committed this change)